project_report/tex/requirements.tex

```diff
@@ -0,0 +1,76 @@
+\section{Requirements}
+
+\subsection{Overall Requirements Specification}
+
+\subsection{Selected Detailed Requirements}
+
+\subsubsection{Functional \& Non-Functional Requirements}
+In table \ref{table:Sup_requirements} you can see a description of Functional \&
+Non-Functional requirements showed in the model of FURPS+. The Model FURPS+ 
+gives a good overview of which Functional \& Non-Functional requirements their is.
+
+FURPS+ is a extend version of FURPS. It extend FURPS with 4 new categories
```

Comment:
Maybe rewrite this paragraph:

FURPS+ is a model for classifying functional and non-functional requirements. The acronym stands for functionality, usability, reliability, performance, and supportability. The '+' sign stands for design constraints, implementation requirements, interface requirements, and physical requirements.